service/entries/router.py

```
@@ -1,25 +1,37 @@
+from fastapi.security import HTTPBearer
 
 from .model.entry import Entry
 from .persistence.entries_store import EntriesStore
+from ..login import token
 
 logger = logging.getLogger(__name__)
 router = APIRouter()
 persistence = EntriesStore()
 
 
+def extract_token(credentials: HTTPAuthorizationCredentials = Depends(HTTPBearer())):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)